readme.md

```diff
@@ -0,0 +1,171 @@
+  "callbackUrl": "https://www.fake.com/",
+  "version": "1",
+  "environment": "PRODUCTION"
+}
+
+```
+
+| parameter    | value                   | Comments                                                                                                     |
+| ------------ | ----------------------- | ------------------------------------------------------------------------------------------------------------ |
+| clientId     | "foo"                   | oauth2 client id of the app                                                                                  |
+| clientSecret | "bar"                   | oauth2 client secret of the app                                                                              |
+| callbackUrl  | "https://www.fake.com/" | oauth2 callback url of the app                                                                               |
+| version      | 1 or 2                  | version number of BlueButton API (v1/v2)                                                                     |
+| environment  | PRODUCTION/SANDBOX      | the target URL:<br>PRODUCTION: https://api.bluebutton.cms.gov<br>SANDBOX: https://sandbox.bluebutton.cms.gov |
+
+## Usages
+
+### Sample Code
+
+Below are psuedo code snippets show SDK used with node express server.
+
+#### Main Usage: Obtain Access Grant and Access Data
+
+```
+
+import express, { Router, Request, Response } from 'express';
+
+const app = express();
+
+const bb = new BlueButton();
+const authData = bb.generateAuthData();
+
+// each associated with current logged in user in real app
+let authToken: AuthorizationToken;
+
+const router = Router();
+
+// auth flow: response with URL to redirect to Medicare.gov beneficiary login
+router.get('/authorize/authurl', bb.generateAuthorizeUrl);
+
+// auth flow: route oauth2 call back
+router.get('/bluebutton/callback', async (req: Request, res: Response) => {
+  authToken = await bb.getAuthorizationToken(authData, req.query.code, req.query.state, req.query.error);
+});
+
+// data flow
+
+// route get EOB requests
+router.get('/data/benefit', async (req: Request, res: Response) => {
+  const data = await bb.getExplanationOfBenefitData(authToken);
+  authToken = data.token;
+  res.json(data.response.data);
+});
+
+// route get Patient requests
+router.get('/data/patient', async (req: Request, res: Response) => {
+  const data = await bb.getPatientData(authToken);
+  authToken = data.token;
+  res.json(data.response.data);
+});
+
+// route get Coverage requests
+router.get('/data/coverage', async (req: Request, res: Response) => {
+  const data = await bb.getCoverageData(authToken);
+  authToken = data.token;
+  res.json(data.response.data);
+});
+
+// route get Profile requests
+router.get('/data/userprofile', async (req: Request, res: Response) => {
+  const data = await bb.getProfileData(authToken);
+  authToken = data.token;
+  res.json(data.response.data);
+});
+
+```
+
+#### Other Usage: Check Scope of Accesss Grant
```

Comment:
Do you think this needs its own section? 

Should we maybe just do a little comment in the section above on how they can pull the scope off the Authorization Token?